src/components/ProfileDashboard.js

```diff
@@ -0,0 +1,466 @@
+                    {missingProgram && <FormHelperText> <strong><p style={{ color: 'red' }}>Please fill this out!</p></strong> </FormHelperText>}
+                </div>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 550, left: "35%" }} >
+                <h5>Interest: </h5>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 540, left: "45%" }} >
+                <div>
+                    {loadProfile ? <h6> {existingInterest} </h6> : <form className={classes.root} noValidate autoComplete="off">
+                        <TextField
+                            id="Interest"
+                            label="Interest"
+                            defaultValue="Enter Interest"
+                            variant="outlined"
+                            style={{ width: "300px" }}
+                            value={interest}
+                            onChange={handleInterest}
+                            inputProps={{ maxLength: 200 }} />
+                    </form>}
+                    {!loadProfile && <FormHelperText> Ex. Coding </FormHelperText>}
+                    {missingInterest && <FormHelperText> <strong><p style={{ color: 'red' }}>Please fill this out!</p></strong> </FormHelperText>}
+                </div>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 660, left: "35%" }} >
+                <h5>Co-op: </h5>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 650, left: "45%" }} >
+                <div>
+                    {loadProfile ? <h6> {existingCoop} </h6> : <form className={classes.root} noValidate autoComplete="off">
+                        <TextField
+                            id="Coop"
+                            label="Coop"
+                            defaultValue="Coop"
+                            variant="outlined"
+                            style={{ width: "300px" }}
+                            value={coop}
+                            onChange={handleCoop}
+                            inputProps={{ maxLength: 200 }} />
+
+                    </form>}
+                    {!loadProfile && <FormHelperText> Role, Company (Ex. None, None) </FormHelperText>}
+                    {missingCoop && <FormHelperText> <strong><p style={{ color: 'red' }}>Please fill this out!</p></strong> </FormHelperText>}
+                </div>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 760, left: "67%" }} >
+                {loadProfile ? <Button variant="outlined" style={{ color: "white", backgroundColor: "red" }} onClick={() => { setLoadProfile(false) }} >Edit</Button> : <Button variant="outlined" style={{ color: "white", backgroundColor: "seagreen" }} onClick={() => { validationCheck() }} >Save</Button>}
+
+                {submission && <FormHelperText> <strong><p style={{ color: 'darkgreen' }}>Your profile has been saved! Please refresh the pages for your changes!</p></strong> </FormHelperText>}
```

Comment:
Instead of doing text confirmation, you should display a success toast message here. 

You can do this by adding this const below your imports:
```
const Alert = React.forwardRef(function Alert(props, ref) {
    return (
        <MuiAlert
            elevation={6}
            ref={ref}
            variant="filled"
            {...props}
            style={{
                position: 'fixed',
                bottom: '20px',
                right: '20px',
                zIndex: 9999
            }}
        />
    );
});
```

Then, use the same conditional rendering condition you have above, but render this component instead:
```
            {submission && (
                <Alert severity="success">
                    Profile successfully edited. 
                </Alert>
            )}
 ```

Reply:
Thanks!